grpc/src/main/java/com/linecorp/armeria/client/grpc/GrpcClientOptions.java

```diff
@@ -129,5 +132,13 @@
             GRPC_CLIENT_STUB_FACTORY = ClientOption.define("GRPC_CLIENT_STUB_FACTORY",
                                                            NullGrpcClientStubFactory.INSTANCE);
 
+    /**
+     * Sets the {@link ClientInterceptor}s to the gRPC client stub.
+     * The last interceptor will have its {@code ClientInterceptor.interceptCall()} called first.
+     */
+    public static final ClientOption<List<ClientInterceptor>>
+            INTERCEPTORS = ClientOption.define("GRPC_CLIENT_INTERCEPTORS",
```

Comment:
nit; no need to line wrap
```suggestion
            INTERCEPTORS = ClientOption.define("GRPC_CLIENT_INTERCEPTORS", ImmutableList.of());
```